### DISCUSSION TITLE
Quand sort la version 6 de bootstrap et aussi ....?

### DISCUSSION THREAD

**Discussion by Esdras-Nsounda** [twbs/bootstrap#40960] [General]
J'aimerai savoir quand la version 6 de bootstrap ? et si cette prochaine version viens t'elle avec un système de suppression de css qui n'est pas utiliser, l'ajout des states comme focus, hover, select, etc...

**Comment by julien-deramond** [comment]
Le développement de la version 6 n'a pas démarré et son contenu n'est pas encore clairement défini.

> et si cette prochaine version viens t'elle avec un système de suppression de css qui n'est pas utiliser

À noter que c'est déjà possible avec des outils comme [PurgeCSS](https://purgecss.com/).

> l'ajout des states comme focus, hover, select, etc...

Ces états sont déjà gérés dans les composants.

**Comment by aniruddhaadak80** [comment]
Hi @Esdras-Nsounda ,  

La version 6 de Bootstrap n'a pas encore de date de sortie officielle annoncée. Concernant les fonctionnalités, il n'y a pas d'informations spécifiques sur un système de suppression de CSS inutilisé ou l'ajout de nouveaux états comme focus, hover, etc. Cependant, la communauté Bootstrap est active et prend souvent en compte les retours des utilisateurs pour les futures mises à jour. Je vous recommande de suivre les canaux officiels de Bootstrap pour les annonces.  

Merci !
